frontend/src/components/PlaylistDropDown.tsx

```
@@ -0,0 +1,71 @@
+import { useEffect, useState } from "react";
+import { fetchPlaylists } from "../services/spotifyApi";
+import type { PlayList, TokenInfo } from "../types/playlists";
+import { SelectedPlaylistInfo } from "./SelectedPlaylistInfo";
+
+export function PlaylistDropDown() {
+  const [playlists, setPlaylists] = useState<PlayList[]>([]);
+  const [options, setOptions] = useState<string>("");
+  const getPlaylists = async (
+    accessToken: string,
+    limit: number,
+    offset: number
+  ) => {
+    try {
+      const playlistResponse = await fetchPlaylists(accessToken, offset, limit);
+      return playlistResponse.items;
+    } catch (error) {
```

maybe throw an error here or log one to the window? its good practice to [fail fast](https://www.martinfowler.com/ieeeSoftware/failFast.pdf) rather than failing silently to immediately see what went wrong. helps with debugging among other things